### ISSUE TITLE
Create a BLT command to check block instance usage across sites

### ISSUE THREAD

**Issue by pyrello** [uiowa/uiowa#2852]
# Story

As a developer I want to be able to see aggregate data about how block components are being used across all sites so that I can use that for better planning how to implement changes to those blocks.

# Proposed solution

TBD

**Comment by richardbporter** [comment]
Yes this would be nice. Having to SSH into each application to run `blt uiowa:multisite:execute` is tedious. A BLT command we run locally that would run remote Drush commands against each application would be nice. Although, resource-intensive commands can be difficult to kill remotely. That has burned us in the past.

There are additional "reports" I would be interested in as well like the total number of sites (and per application), including sitenow version. There might be some intersection with things like that and this issue.
